test/check_performance.rb

```diff
@@ -39,577 +40,18 @@ class Ticket < ActiveRecord::Base
-    database: ars_test_shard_62_replica
-
-  test_shard_63:
-    <<: *MYSQL
-    database: ars_test_shard_63
-  test_shard_63_replica:
-    <<: *MYSQL
-    database: ars_test_shard_63_replica
-
-  test_shard_64:
-    <<: *MYSQL
-    database: ars_test_shard_64
-  test_shard_64_replica:
-    <<: *MYSQL
-    database: ars_test_shard_64_replica
-
-  test_shard_65:
-    <<: *MYSQL
-    database: ars_test_shard_65
-  test_shard_65_replica:
-    <<: *MYSQL
-    database: ars_test_shard_65_replica
-
-  test_shard_66:
-    <<: *MYSQL
-    database: ars_test_shard_66
-  test_shard_66_replica:
-    <<: *MYSQL
-    database: ars_test_shard_66_replica
-
-  test_shard_67:
-    <<: *MYSQL
-    database: ars_test_shard_67
-  test_shard_67_replica:
-    <<: *MYSQL
-    database: ars_test_shard_67_replica
-
-  test_shard_68:
-    <<: *MYSQL
-    database: ars_test_shard_68
-  test_shard_68_replica:
-    <<: *MYSQL
-    database: ars_test_shard_68_replica
-
-  test_shard_69:
-    <<: *MYSQL
-    database: ars_test_shard_69
-  test_shard_69_replica:
-    <<: *MYSQL
-    database: ars_test_shard_69_replica
-
-  test_shard_70:
-    <<: *MYSQL
-    database: ars_test_shard_70
-  test_shard_70_replica:
-    <<: *MYSQL
-    database: ars_test_shard_70_replica
-
-  test_shard_71:
-    <<: *MYSQL
-    database: ars_test_shard_71
-  test_shard_71_replica:
-    <<: *MYSQL
-    database: ars_test_shard_71_replica
-
-  test_shard_72:
-    <<: *MYSQL
-    database: ars_test_shard_72
-  test_shard_72_replica:
-    <<: *MYSQL
-    database: ars_test_shard_72_replica
-
-  test_shard_73:
-    <<: *MYSQL
-    database: ars_test_shard_73
-  test_shard_73_replica:
-    <<: *MYSQL
-    database: ars_test_shard_73_replica
-
-  test_shard_74:
-    <<: *MYSQL
-    database: ars_test_shard_74
-  test_shard_74_replica:
-    <<: *MYSQL
-    database: ars_test_shard_74_replica
-
-  test_shard_75:
-    <<: *MYSQL
-    database: ars_test_shard_75
-  test_shard_75_replica:
-    <<: *MYSQL
-    database: ars_test_shard_75_replica
-
-  test_shard_76:
-    <<: *MYSQL
-    database: ars_test_shard_76
-  test_shard_76_replica:
-    <<: *MYSQL
-    database: ars_test_shard_76_replica
-
-  test_shard_77:
-    <<: *MYSQL
-    database: ars_test_shard_77
-  test_shard_77_replica:
-    <<: *MYSQL
-    database: ars_test_shard_77_replica
-
-  test_shard_78:
-    <<: *MYSQL
-    database: ars_test_shard_78
-  test_shard_78_replica:
-    <<: *MYSQL
-    database: ars_test_shard_78_replica
-
-  test_shard_79:
-    <<: *MYSQL
-    database: ars_test_shard_79
-  test_shard_79_replica:
-    <<: *MYSQL
-    database: ars_test_shard_79_replica
-
-  test_shard_80:
+  <% NUMBER_OF_SHARDS.times do |shard_id| %>
+  test_shard_<%= shard_id %>:
     <<: *MYSQL
-    database: ars_test_shard_80
-  test_shard_80_replica:
+    database: ars_test_shard<%= shard_id %>
+  test_shard_<%= shard_id %>_replica:
     <<: *MYSQL
-    database: ars_test_shard_80_replica
+    database: ars_test_shard<%= shard_id %>_replica
```

Comment:
I think we missed an underscore here:
```suggestion
    database: ars_test_shard_<%= shard_id %>_replica
```

Reply:
Nope, I was wrong.